src/cr/cube/matrix/measure.py

```diff
@@ -565,6 +566,7 @@ def p_vals(self):
             [
                 [
                     _PairwiseSignificaneBetweenSubvariablesHelper(
+                        self._cube_measures.weighted_cube_counts.weighted_counts,
```

Comment:
what if `weighted_counts`  is `None`? This is an optional cube measure and if it doesn't exists its value is None. Do we have a test that exercises this case?